staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -563,6 +566,8 @@ func (qs *queueSet) dispatchLocked() bool {
 	qs.totRequestsWaiting--
 	qs.totRequestsExecuting++
 	queue.requestsExecuting++
+	metrics.ChangeRequestsInQueues(qs.qCfg.Name, request.fsName, -1)
```

Comment:
https://prometheus.io/docs/concepts/metric_types/#counter

> A counter is a cumulative metric that represents a single monotonically increasing counter whose value can only increase or be reset to zero on restart

it says the counter should only increase monotonically

Reply:
If I understand correctly, the problem here was the comment on `metrics.ChangeRequestsInQueues` which used the term "count" in an informal way --- which is confusing because "count" is a technical term in these metrics.  I changed the comment to use the term "gauge" instead.

Reply:
oh nevermind.. it's a gauge